app/templates/questionnaire.html

```diff
@@ -63,10 +63,11 @@ <h1 class="alert__title venus">This page has {{all_errors|length}} errors</h1>
 
       <button class="btn btn--primary qa-btn-submit venus" type="submit" name="action[save_continue]">{% block submit_button_text %}Save and continue{% endblock %}</button>
       <br/>
+      {% if state.group_instance > 0 or not block.first_in_schema() %}
```

Comment:
I don't think adding this sort of logic to the schemas is the way to go for resolving this issue. The previous location is calculated before rendering the template and presumably should be none for the first pages on census schemas, so isn't it easier just to check it isn't None?